src/avt/Plotter/vtk/vtkVisItAxisActor2D.C

```diff
@@ -547,13 +552,6 @@ void vtkVisItAxisActor2D::BuildAxis(vtkViewport *viewport)
   bool propsModified = (this->TitleTextProperty->GetMTime() > this->BuildTime) ||
                        (this->LabelTextProperty->GetMTime() > this->BuildTime);
 
-  if (this->GetMTime() < this->BuildTime &&
-      viewport->GetMTime() < this->BuildTime &&
-      !propsModified)
-    {
-    return; //already built
-    }
-
```

Comment:
Note: This early out was the reason the axis lines would end up skewed. Even if the axis properties did not change -- the viewport may have, and we need to rebuild the axis with respect to the new viewport.